apps/demo-typegen/src/demo.test.ts

```diff
@@ -55,28 +55,28 @@ describe('ExampleContract', () => {
 
     // You can also make a call using the factory
     // #region typegen-demo-contract-factory-connect
-    // #context import { DemoContractAbi__factory } from './types';
+    // #context import { DemoContract } from './typegend';
 
-    const contractInstance = DemoContractAbi__factory.connect(contractId, wallet);
+    const contractInstance = new DemoContract(contractId, wallet);
     const call2 = await contractInstance.functions.return_input(1337).call();
     const { value: v2 } = await call2.waitForResult();
     // #endregion typegen-demo-contract-factory-connect
     expect(v2.toHex()).toBe(toHex(1337));
   });
 
-  it('deployContract method', async () => {
+  it('deploy method', async () => {
     using launched = await launchTestNode();
 
     const {
       wallets: [wallet],
     } = launched;
 
     // #region typegen-demo-contract-factory-deploy
-    // #context import { DemoContractAbi__factory } from './types';
-    // #context import bytecode from './types/DemoContractAbi.hex';
+    // #context import { DemoContract } from './typegend';
+    // #context
```

Comment:
Why is this `#context` here?

Reply:
👀  https://github.com/FuelLabs/fuels-ts/pull/2824/commits/f232d159b8c3681162c06e60a96c5fdcd6c3aa15